XaiValidatePublicApiDefinition.yml

```diff
@@ -4315,6 +4125,19 @@ components:
                                     type: number
                                     description: Applicant ownership percentage
                                       for property.
+                                  HomeLoanInterestRate:
+                                    type: number
+                                    description: Declared home loan interest rate.
+                                  HomeLoanTerm:
+                                    type: number
+                                    description: Declared home loan term.
+                                  HomeLoanRepaymentType:
+                                    type: string
+                                    description: Declared home loan repayment type.
+                                    enum:
+                                    - PI
+                                    - IO
+                                    - LOC
```

Comment:
@rjshanahan could we clarify what LOC is?

Reply:
@jync `LOC` is for Line of Credit home loans - rare but they're still out there, and required different repayment calc treatment